aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/PersistentMutationOutbox.java

```diff
@@ -321,6 +322,7 @@ private Completable handleIncomingCreate() {
          * Determine which action to take when the incoming mutation type is {@linkplain PendingMutation.Type#UPDATE}.
          * @return A completable with the actions needed to resolve the conflict
          */
+        @SuppressWarnings("unchecked") // cast SerializedModel to Model
```

Comment:
```suggestion
        @SuppressWarnings("unchecked") // cast to SerializedModel
```

Reply:
As discussed offline, moved the annotation inline